docs/source/api_reference/pipeline_graph.rst

```diff
@@ -1 +1,2 @@
 .. automodule:: easylink.pipeline_graph
+    :show-inheritance:
```

Comment:
Given that this repo's API docs are really only for developers, I figured we should turn this on